src/features/usage-consolidation-accordion/usage-consolidation-content-package.js

```diff
@@ -0,0 +1,142 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  useIntl,
+  FormattedMessage,
+} from 'react-intl';
+import getSymbolFromCurrency from 'currency-symbol-map';
+
+import {
+  MultiColumnList,
+  Button,
+  Dropdown,
+  DropdownButton,
+  DropdownMenu,
+} from '@folio/stripes/components';
+
+import { costPerUse as costPerUseShape } from '../../constants';
+
+const propTypes = {
+  costPerUseData: costPerUseShape.CostPerUseReduxStateShape.isRequired,
+  year: PropTypes.string.isRequired,
+};
+
+const UsageConsolidationContentPackage = ({
+  costPerUseData,
+  year,
+}) => {
+  const intl = useIntl();
+  const {
+    isLoaded,
+    isFailed,
+    data,
+  } = costPerUseData;
+
+  if (!isLoaded && !isFailed) {
+    return null;
+  }
+
+  if (isFailed) {
+    return (
+      <div data-test-usage-consolidation-error>
+        <FormattedMessage id="ui-eholdings.usageConsolidation.summary.error" />
+      </div>
+    );
+  }
+
+  const currency = data?.attributes?.parameters?.currency;
+  const currencySymbol = getSymbolFromCurrency(currency);
+
+  const {
+    cost,
+    costPerUse,
+    usage,
+  } = data?.attributes?.analysis;
+  const noCostPerUseAvailable = !cost && !costPerUse && !usage;
+
+  const formatCost = (value) => `${currencySymbol || ''}${value} (${currency})`;
```

Comment:
Just suggestion
```suggestion
  const currencySymbol = getSymbolFromCurrency(currency) || '';

  const {
    cost,
    costPerUse,
    usage,
  } = data?.attributes?.analysis;
  const noCostPerUseAvailable = !cost && !costPerUse && !usage;

  const formatCost = (value) => `${currencySymbol}${value} (${currency})`;
```
It makes `formatCost` little bit more readable